src/constants/boxShadows/index.ts

```diff
@@ -2,6 +2,7 @@
  * Note: Each Category is exported separately to be rendered in storybook:
  */
 
+// @ts-ignore
```

Comment:
I think your branch with the unpushed changes should fix these kinds of issues.

Reply:
Oh this was temporary my focus was to make the build work. 
Pushing `utils` convertion now. Anyway I was testing just to make `1` ONE freaking constant to work `ANIMATION.defaultTiming` that didnt even use any utils. to not avail. I will put here the solution I reach but I am NOT happy with it